### DISCUSSION TITLE
Using two route groups with with nested dynamic routes.

### DISCUSSION THREAD

**Discussion by supra-jatinjainsaraf** [vercel/next.js#57056] [App Router]
I am having difficulty configuring the layout for different route groups. The **_/f_** route is not accessible and redirects to an error page. I am using the next 13.5 App router and have attached a screenshot of my routing structure.

![Screenshot from 2023-10-19 18-50-48](https://github.com/vercel/next.js/assets/142773930/f7697ae1-2f45-421f-9788-bdb7e4ccfb9b)

**Comment by 1010nishant** [comment]
yes, I am also facing the same issue.
the error is 
Error: The default export is not a React Component in page: (that route)

**Comment by anilcan-kara** [comment]
The point is just use single dynamic page [network]
and use single layout.js at /app/[network]/layout.js
then split 2 different page.js in that layout.js with if conditions,

here is sample /app/[network]/layout.js
```jsx
export default async function RootLayout({ home, txDetails, params: { network } }) {
    const yourConditions1 = 3+5 === 8 ? true : false;
    const yourConditions2 = network === 'test' ? true : false;
    
    return (
        <html>
            <body>
                    {yourConditions1 && yourConditions2 && home || txDetails}
            </body>
        </html>
    )
}
```


2 different pages should be placed in /app/[network]/@home/page.js and /app/[network]/@txDetails/page.js
they shouldn't has to layout.js with page.js, only page.js
